src/components/specimen/steps/AnnotateSteps.tsx

```diff
@@ -42,21 +42,31 @@ const AnnotateSteps = (props: Props) => {
 
     /* Dummy Annotation for Showcase */
     const dummyAnnotation: Annotation = {
-        id: '20.5000.1025/dummy',
-        version: 1,
-        type: 'Annotation',
-        motivation: 'commenting',
-        target: {
-            id: specimen.digitalSpecimen['ods:id'],   
-            type: 'digitalSpecimen',
-            indvProp: 'ods:specimenName'
+        "ods:id": '20.5000.1025/dummy',
```

Comment:
maybe make the prefix TEST or something else

Reply:
Changed to TEST